guide/getting-started/why-bitcoin-is-unique.md

```diff
@@ -35,16 +35,7 @@ Illustration sources
 
 # Why Bitcoin is unique
 
-The internet has changed the world. Barriers to global communication and the exchange of information are reduced to almost nothing. Bitcoin adds another important dimension to this and brings us a natively digital method for storing and transferring value.
-
-<!--
-The status quo is that currencies are created and managed by governments and online payments are routed through the payment networks of large corporations. Government authority ensures that currencies are recognized and backed, while payment providers stay on top of fraud, prevent double-spending, provide security and various other benefits and services. This highly complex system works well enough for most people.
-
-Bitcoins unique and clever combination of different technologies has managed to create both a currency and a payment network that can be trusted without government or corporate authorities. This innovation does not only open up global payments to anyone with a smartphone or computer, it also allows for storing value outside of traditional economic systems. This does not mean that bitcoin will replace existing currencies. Instead, it can complement them by better serving audiences and use cases that are typically difficult and expensive, as well as opening up completely new possibilities.
-
-We have already hinted at some of the unique properties of Bitcoin, but let’s go through them one by one, seen through the lens of Bitcoin as money.
-
--->
+The Internet lets people anywhere instantly exchange ideas and information with people everywhere. Bitcoin takes this further by giving everyone a baked in method for storing and transferring value online.
```

Comment:
Is this clear for non-English speakers?

Reply:
I don't think so, what's the alternative we can use?

Reply:
"...Bitcoin takes this further by giving everyone an internet native method for storing and transferring value..."

Re-referring back to the internet which was introduced in the prior sense more sense to me.

Reply:
@sqcken what do you think?

Reply:
```suggestion
The Internet lets people anywhere instantly exchange ideas and information with people everywhere. Bitcoin takes this further by giving everyone a built-in method for storing and transferring value online.
```